build/build-usb.bash

```diff
@@ -8,6 +8,9 @@ else
     echo sudo mount --bind $HARVEY_GOLANG ../sys/go
     sudo mount --bind $HARVEY_GOLANG ../sys/go
     ls -l ../sys/go
+    pushd ../sys/src/cmd/go
```

Comment:
(cd ../sys/src/cmd/go && sh fetchrepos.sh)
it's just a lot safer, a missing pushd/popd can make life awful.